Documentation/install/upgrade.rst

```diff
@@ -316,6 +316,24 @@ Annotations:
 1.7 Upgrade Notes
 -----------------
 
+1.7.13 Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new flag ``--egress-multi-home-ip-rule-compat`` was introduced to allow
+specifying whether to maintain compatibility with the older ENI mode datapath.
+It defaults to ``false`` meaning it will instruct Cilium to **not** maintain
+compatibility and perform a migration to the new ENI datapath, upon Cilium
+upgrade. This operation will modify the routing table to fix a potential
+collision.
+
+If the flag is ``true`` upon upgrade, Cilium will **not** perform a migration.
+
+If the migration was already completed and a downgrade to an older version of
+Cilium is desired, you must first set the flag to ``true`` and then restart
+Cilium so that it can rollback the migration. This ensures that the new ENI
+datapath is rolled back to a state where it is compatible with the older
+version of Cilium.
+
```

Comment:
```suggestion
If the migration was already completed and you wish to downgrade to
v1.7.12 or earlier, you must first set the flag to ``true`` and then restart
Cilium so that it can rollback the migration. This ensures that the new ENI
datapath is rolled back to a state where it is compatible with the older
version of Cilium. Once this has migrated back to the compatibility mode,
further downgrade to v1.7.12 or earlier can be performed without connectivity
disruption to active pods.

```